TileStache/Goodies/VecTiles/transform.py

```diff
@@ -291,44 +291,45 @@ def road_classifier(shape, properties, fid, zoom):
 def road_sort_key(shape, properties, fid, zoom):
     # Note! parse_layer_as_float must be run before this filter.
 
-    # Calculated sort value is in the range 0 to 39
-    sort_val = 0
+    floor = 300
+    ceiling = 385
+    sort_val = floor
 
-    # Base layer range is 15 to 24
     highway = properties.get('highway', '')
     railway = properties.get('railway', '')
     aeroway = properties.get('aeroway', '')
     aerialway = properties.get('aerialway', '')
     service = properties.get('service')
 
-    is_railway = railway in ('rail', 'tram', 'light_rail', 'narrow_guage', 'monorail')
+    is_railway = railway in (
+        'rail', 'tram', 'light_rail', 'narrow_guage', 'monorail')
 
     if highway == 'motorway':
-        sort_val += 24
+        sort_val += 44
     elif is_railway:
-        sort_val += 23
+        sort_val += 43
     elif highway == 'trunk':
```

Comment:
Should also include the following Natural Earth values:
`or type == 'Secondary Highway'`